### ISSUE TITLE
Create a collapsible header with icon

### ISSUE THREAD

**Issue by IreneStr** [Yoast/yoast-components#654]
Create a collapsible header with icon. This header will be used both in the sidebar, as well as in the metabox. This means its width should be flexible. We already have a collapsible header without icon, so please use that one as a basis. The color of the icon and the icon itself should be able to be changed based on passed props.

## Sidebar
<img width="396" alt="schermafbeelding_2018-07-10_om_11_45_47" src="https://user-images.githubusercontent.com/17744553/42502645-ed92937e-8436-11e8-8b3b-c1f3ffb0b90d.png">

## Metabox
<img width="416" alt="schermafbeelding_2018-07-10_om_11_45_53" src="https://user-images.githubusercontent.com/17744553/42502664-fb353f9a-8436-11e8-8f13-0313f0718aae.png">

## SVG icons
[Zip with smiley SVGs](https://github.com/Yoast/yoast-components/files/2180173/score.smileys.zip). The grey icon is a bullet without a face.
Colors:
#008800 (green)
#f5c819 (yellow)
#dc3232 (red)
#a0a5aa (grey)

**Comment by hedgefield** [comment]
The associated color variables are:
`$color_bad`
`$color_ok`
`$color_good`
`$color_score_icon`

**NOTE** that in [our colors scss](https://github.com/Yoast/yoast-components/blob/develop/style-guide/_colors.scss) these variables don't have the colors that are specified in the design, they're slightly off:

color_good uses `palette_green_light` now, that should be `palette_green_medium_light`
color_ok uses `palette_orange_light` now, that should be `palette_yellow`
color_score_icon uses `palette_grey_medium_dark` now, that should be `palette_grey_medium`

While these colors off a better colorblind contrast than the existing ones, I don't just want to pull the trigger on changing them as it will impact all the bullets in the plugin. I'll open a separate issue for this to do another a11y review. So the icons can be implemented now, just with the added context that the colors need some work in parallel to this.

**Comment by hedgefield** [comment]
After the a1yy review we tweaked the colors a little bit, and I made versions of the icons that have the colors baked in. [Here they are](https://github.com/Yoast/yoast-components/files/2188530/color_smileys.zip).

**Comment by afercia** [comment]
@hedgefield re:the SVG icons, we'd need to always generate the SVG code in the same way. All the other icons used in the yoast-components have a `viewBox="0 0 1792 1792"` while these new ones have a `viewBox="0 0 496 512"`.

While, with some code acrobatics, it is possible to pass down `viewBox` as a prop through a long chain of components, I'd like to avoid that 🙂 The best option would be having all the icons with the same viewBox. It is possible to regenerate them using the same standard viewBox size?

Depending on the intended use, there are also other things to strip out from the SVG code but that can be done manually. Thanks!

**Comment by hedgefield** [comment]
I'd be down for making all the svgs the same size. Is it possible to generate a list somewhere which those are?

SVG optimization is a weird process and almost no image editor does it right, so I'm not too sure about the rest of the things.

**Comment by afercia** [comment]
Thanks! The ones we're using right now in the yoast-components are the ones listed in https://github.com/Yoast/yoast-components/blob/develop/composites/Plugin/Shared/components/SvgIcon.js

I'm in the process of adding the new 3 seo-score ones of course.

**Comment by hedgefield** [comment]
I made an issue for it on our design repo 👍https://github.com/Yoast/design/issues/174

**Comment by Xyfi** [comment]
This issue gets fixed by #653. Closing in favor of that issue.
